src/python/ripe_demo/static/js/main.js

```diff
@@ -4,7 +4,7 @@
  */
 var FACES = ["side", "top", "front"];
 
-window.onload = function() {

```

Comment:
@3rdvision something smells fishy. Make sure you have the latest dependencies, this seems like an old prettier config.

Reply:
Even after removing and reinstalling modules and even upgrading, it formats this way after `npm run pretty`... wierd! 🐟 
Reverting to original.